users/tests.py

```diff
@@ -0,0 +1,62 @@
+# from django.test import TestCase
+# from asyncio import FastChildWatcher
+import pytest
+from django.core.exceptions import ValidationError
+from . import models
+
+
+PHONE_NUM = "0528305110"
+EMAIL = "yael@gmail.com"
+PASSWORD = "password"
+
+
+@pytest.fixture
+def user0():
+    return models.User(email=EMAIL, phone_number=PHONE_NUM, password=PASSWORD)
+
+
+def test_new_user(user0):
+    assert user0.email == EMAIL
+    assert user0.phone_number == PHONE_NUM
+    assert user0.name == 'User'
+    assert user0.profile_pic == 'profile_pic.svg'
+    assert user0.password == PASSWORD
+
+
+@pytest.mark.django_db
+def test_failed_creation_user1():
+    try:
+        user = models.User(email="ilan@gmail.com", phone_number="090398a", password=PASSWORD)
+        user.full_clean()
+    except ValidationError:
+        assert True  # The user was not! created successfully
+    else:
+        assert False  # The user was created successfully
+
+
+@pytest.mark.django_db
+def test_failed_creation_user2():
```

Comment:
same ^^
```suggestion
def test_invalid_email_address():
```
maybe it will be even better to use the parametrize as you can see an example in:
https://github.com/redhat-beyond/ClassRater/blob/main/homepage/tests/add_review_search_tests.py#L16
instead of repeating the kind of the same tests in lines #27-34 and #38-45

Reply:
We have combind the two test functions- email and phone number into one test function with parametrize.